pages/web-literacy/MakeLinksFromWebLitSkills.jsx

```diff
@@ -0,0 +1,21 @@
+var React = require('react');
+var weblitcontent = require('./weblitcontent');
+var webLitSkillsContent = weblitcontent.webLitSkills;
+var ReactRouter = require('react-router');
+var Link = ReactRouter.Link;
+
+var MakeLinksFromWebLitSkills = function(webLitSkills) {
+  return webLitSkills.map(function(webLitSkill, index) {
+    return (
+      <span className="comma-separated-links" key={webLitSkill}>
```

Comment:
do we need the span? Can we not just set these props directly on the link itself?